addon/components/ui-kind.js

```diff
@@ -1 +0,0 @@
-export { default } from "ui-base-theme/components/ui-kind";
```

Comment:
don't need to reexport this anymore; ui-tomato-theme can autogenerate components from ui-base-theme now